lib/generator/frameThroughputDctGenerator.py

```diff
@@ -59,7 +59,25 @@ def get_frame_throughput(self, result_list, input_image_list):
 
             # calculate viewport variations between start point and end point
             # based on variation to determine if frame is freeze and result in frame throughput related values
-            start_target_fp = input_image_list[image_fn_list[start_event_index]][CropRegion.VIEWPORT]
+            start_visual_event = dict()
+            end_visual_event = dict()
+            ft_compare_area = CropRegion.VIEWPORT
+            for _, event_list in self.visual_event_points.items():
+                if not start_visual_event:
+                    start_visual_event = self.get_event_data_in_result_list(event_list, self.EVENT_START)
+                if not end_visual_event:
+                    end_visual_event = self.get_event_data_in_result_list(event_list, self.EVENT_END)
+            if start_visual_event['search_target'] == end_visual_event['search_target'] or \
```

Comment:
```
attribute_needed = set(['x', 'y', 'w', 'h'])
(set(start_visual_event) & set(end_visual_event) & attribute_needed == attribute_needed)
```
If this is `true`, then
`start_visual_event.get('x', None) and end_visual_event.get('x', None) and start_visual_event.get('y', None) and end_visual_event.get('y', None) and start_visual_event.get('w', None) and end_visual_event.get('w', None) and start_visual_event.get('h', None) and end_visual_event.get('h', None)` is `true`